compositions/cluster-k8s/big-blocks/tc-001-val-large-txs/40-vals-180kb-pfd/256mib-0ms.toml

```diff
@@ -5,19 +5,21 @@
 [global]
   plan = "celestia"
   case = "001-val-large-txs"
-  total_instances = 40
+  total_instances = 42
   builder = "docker:generic"
   runner = "cluster:k8s"
   disable_metrics = false
-
-[global.run.test_params]
-  execution-time = "10"
-  latency = "0"
-  bandwidth = "256Mib"
-  validator = "40"
-  persistent-peers = "10"
-  submit-times = "10"
-  msg-size = "180000"
+  [global.run]
+    artifact = ""
+    [global.run.test_params]
+      bandwidth = "256Mib"
+      execution-time = "10"
+      latency = "0"
+      msg-size = "160000"
```

Comment:
Non-Blocking: I wonder if we can fine tune these values to account for padding to result exactly in 180kb.